src/main/java/gov/nasa/pds/tools/validate/ValidationTarget.java

```diff
@@ -189,8 +190,7 @@ public String toString() {
   @Override
   public int hashCode() {
     if (knownHashCode == 0) {
-      String combined = location + ":" + (type == null ? "" : type.toString());
-      knownHashCode = combined.hashCode();
+      knownHashCode = location.hashCode();
```

Comment:
Is there ever a risk that `location.hashCode()` would be `0`, and thus re-compute the hash code?

I guess it doesn't really matter, though … hash codes aren't tremendously expensive

Reply:
Yes. Not in infinite loop but will recompute 0 every time. I left the knownHashCode for nostalgia. Besides location is a string and I bet Java keeps a static value of it around just like it does the string since they are immutable. Really should just return location.hashCode() and not try to buffer it but I am a nostalgic person.

Reply:
Hey, a little nostalgia now and then is just fine 😌